packages/@guidesmiths/cuckoojs-schematics/src/helm/schema.json

```diff
@@ -4,6 +4,23 @@
   "title": "helm Options Schema",
   "type": "object",
   "properties": {
+    "helmType": {
+      "type": "string",
+      "description": "type of helm chart",
+      "x-prompt": "Which type of helm chart you want to use?",
+      "enum": ["generic", "ob"],
+      "default": "generic",
+      "items": [
+        {
+          "value": "generic",
+          "label": "Generic - basic helm chart for a generic service"
+        },
+        {
+          "value": "ob",
+          "label": "OB Cluster - for deploying on a cluster with Traefik and CertManager (OB internal projects)"
+        }
```

Comment:
This makes the library opinionated about specific OB helm configuration. I'm not sure if we should do it this way, because we try to make the library as much agnostic as possible.

@sergioCB95, perhaps we can discuss this internally to see if we can find an alternative for this.

Reply:
Agree, I don't like much the idea of adding some "OB" specific functionality. Let's have an internal chat on this.